graphics/include/ignition/common/ColladaExporter.hh

```diff
@@ -53,6 +70,11 @@ namespace ignition
           const std::string &_filename, bool _exportTextures,
           const std::vector<math::Matrix4d> &_submeshToMatrix);
 
+      public: void Export(const Mesh *_mesh,
```

Comment:
It looks like the function above is also missing docs; it would be nice to document both.

Reply:
documented both functions 1b7602b